src/hazelcore/ZTyp.re

```diff
@@ -142,6 +142,7 @@ and move_cursor_left_zoperand =
   | CursorT(OnOp(_) | OnText(_), _) => None
   | CursorT(OnDelim(k, After), ty) =>
     Some(CursorT(OnDelim(k, Before), ty))
+  | CursorT(OnDelim(0, Before), _) => None
   | CursorT(OnDelim(_, Before), Hole | Unit | Int | Float | Bool) => None
```

Comment:
I think this is redundant now because for these forms there is only delim 0

Reply:
oh but maybe need it to satisfy exhaustiveness checker?

Reply:
@cyrus- The line you highlighted (line 145) is used to handle on-the-left-edge case for everyone. The next line (line 146) is there to satisfy the exhuastiveness checker because even though `Hole`, `Unit` etc can only take a delimitor of `0`, the compiler is unaware of that.